tests/unit/configuration/test_pricing.py

```
@@ -0,0 +1,134 @@
+                "cost": {},
+            },
+            "no-cost": {
+                "name": "No Cost Model",
+            },
+        },
+    },
+}
+
+
+@pytest.fixture(autouse=True)
+def _clear_cache():
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_clear_cache`

(ANN202)